client/src/components/Modals/Modal.js

```diff
@@ -31,18 +32,30 @@ const BadgeContainer = styled(RemoveContainer)`
 `;
 
 export default function DictonaryList({ onClick }) {
-  return (
-    <>
-      <ModalContainer>
-        <RemoveContainer>
-          <button onClick={() => onClick()}>
-            <RemoveIcon />
-          </button>
-        </RemoveContainer>
-        <BadgeContainer>
-          <Badge>Hallo</Badge>
-        </BadgeContainer>
-      </ModalContainer>
-    </>
-  );
+  let words = [];
+  words = useFetch('http://localhost:7100/api/auth');
+
+  console.log(words);
+  console.log(words.vocabulary);
+
+  if (!words.vocabulary) {
+    return <></>;
```

Comment:
instead of a fraction, you could return `null` (more common if you don't want to return anything)

Reply:
if I set it to null, I get a typerrro: Cannot read property of null